julia/cxx/README.md

```diff
@@ -0,0 +1,9 @@
+# Simple C++ kernel implementations
+
+This directory contains simple (straightforward) C++ implementations
```

Comment:
This code might be more valuable as part of the CI process (e.g. as a pipeline the computes the CPU version of the GPU logic, and compares them.   I realize not all the tooling is in place yet, but we should keep this in mind for when it is.  Otherwise I worry we'll forget about keeping this code upto date with any kernel changes we might make later.